src/typeIndex/typeIndexLogic.ts

```diff
@@ -155,12 +155,32 @@ export function createTypeIndexLogic(store, authn, profileLogic, utilityLogic):
 
     async function getScopedAppsFromIndex(scope: TypeIndexScope, theClass: NamedNode | null): Promise<ScopedApp[]> {
         const index = scope.index
+        const results: ScopedApp[] = []
         const registrations = store.statementsMatching(null, ns.solid('instance'), null, index)
             .concat(store.statementsMatching(null, ns.solid('instanceContainer'), null, index))
             .map(st => st.subject)
         const relevant = theClass ? registrations.filter(reg => store.any(reg, ns.solid('forClass'), null, index)?.sameTerm(theClass))
             : registrations
-        const directInstances = relevant.map(reg => store.each(reg, ns.solid('instance'), null, index).map(one => sym(one.value))).flat()
+
+        for (const reg of relevant) {
```

Comment:
Is this new logic covered by tests?

Reply:
I could not understand what the `one => sym(one.value)` was for. Any ideas? It seems `sym`  replaced a NamedNode with a plain JS object.  Yes, if you take out the filter by using registrations instead of relevant then tests fail as you get a lot more results.